### DISCUSSION TITLE
Some ISA Arch's instruction bounary not divided  by Byte

### DISCUSSION THREAD

**Discussion by baikaishiuc** [Vector35/binaryninja-api#5073] [API]
In recently , I reverse engineering a vmp protected android app.

I found the vmp one instruction opcode and operand is 6bit length, and so the vmp instruction length is 6bit * N

I read the binary ninja arch manual : https://github.com/Vector35/Z80

It seems now BN only support byte boundary instruction analysis, like arm64 (4 byte), and amd64(up to 15 bytes)

How can I do to resolve this problem ?

**Comment by plafosse** [comment]
Yes, this is a known deficiency. We hope to address this at some point but its not on the immediate roadmap. I just created this issue to track this: https://github.com/Vector35/binaryninja-api/issues/5174
